### PR TITLE
use ansible as the single point of truth

### PR DESCRIPTION
**Pull request by robertkowalski**
As `configuration-production.js` is not checked in, looking at a
diff is not possible.

Managing our whole config from ansible using source control makes
it easier for us to maintain our software.

### CODE REVIEW

File: provisioning/loadbalancer.yml [hunk 1]
```diff
@@ -15,7 +15,6 @@
 
     - name: enable haproxy for ubuntu
       template: src=templates/haproxy.j2 dest=/etc/default/haproxy backup=yes
-      notify: restart haproxy
```
Comment (robin-drexler) [inline provisioning/loadbalancer.yml:18]: I'd rather keep that. If you change the content of this file, the service needs to be restarted. Or am I missing something?

Comment (robertkowalski) [inline provisioning/loadbalancer.yml:18]: ah ok!

**Comment by robertkowalski** [comment]
fixed and rebased!

**Comment by robin-drexler** [comment]
lgtm, very cool

**Pull request merged**